### PR TITLE
[TimeLock] Docs: Steps for Namespace Changes

### PR DESCRIPTION
**Pull request by jeremyk-91**
**Goals (and why)**:
- Explain how to switch timelock client namespaces, as part of internal documentation for doing this for AtlasDb products in general

**Implementation Description (bullets)**:
- Write the docs.

**Testing (What was existing testing like?  What have you done to improve it?)**:
The docs build!

**Concerns (what feedback would you like?)**:
- Are there enough warnings here?

**Where should we start reviewing?**: `service-management.rst`

**Priority (whenever / two weeks / yesterday)**: this week

@amandachow for SA

### CODE REVIEW

File: docs/source/services/timelock_service/service-management.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+When a TimeLock client wants to switch its namespace (while preserving its data), care must be taken to ensure that the
+old timestamp bound is carried over to the new namespace. This is to preserve the guarantee of the timestamp service
+that timestamps must be increasing.
+
+Step 1: Shut Down Clients
+-------------------------
+
+Shut down all instances of the TimeLock client.
+This is done to ensure that the timestamp bound does not advance further while we are doing the other step.
```
Comment (gmaretic) [inline docs/source/services/timelock_service/service-management.rst]: nit: steps

Comment (gmaretic) [reply docs/source/services/timelock_service/service-management.rst]: also "of the TimeLock client for which you wish to change the namespace". I can see people turning off TimeLock itself by accident

File: docs/source/services/timelock_service/service-management.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+==================
+
+Namespace Changes
+=================
+
+.. danger::
+
+   Be careful when changing namespaces - doing this process incorrectly may result in **SEVERE DATA CORRUPTION**.
+   This guide also **ONLY** covers the steps for migrating a TimeLock client - AtlasDB clients, separately, must also
+   migrate the data in their key-value services to the new namespace.
```
Comment (gmaretic) [inline docs/source/services/timelock_service/service-management.rst]: Link to docs?

File: docs/source/services/timelock_service/service-management.rst (hunk 1 above)
Comment (jeremyk-91) [reply docs/source/services/timelock_service/service-management.rst]: Sure, put in both changes. Also added another line telling people to explicitly `Leave TimeLock running; you'll need it for the remaining steps.`

File: docs/source/services/timelock_service/service-management.rst (hunk 2 above)
Comment (jeremyk-91) [reply docs/source/services/timelock_service/service-management.rst]: This link is on internal docs. I've added a line telling external and internal users steps they should do.

**Pull request merged**